app/Http/Controllers/Apis/Protected/Summit/OAuth2SummitEventsApiController.php

```diff
@@ -800,7 +800,10 @@ public function deleteEvent($summit_id, $event_id)
             $summit = SummitFinderStrategyFactory::build($this->repository, $this->resource_server_context)->find($summit_id);
             if (is_null($summit)) return $this->error404();
 
-            $this->service->deleteEvent($summit, $event_id);
+            $current_member = $this->resource_server_context->getCurrentUser();
```

Comment:
@romanetar why  is this condition is added ? if there is no current user then is a service account and should be allowed